src/ontology/mondo-edit.obo

```diff
@@ -509817,29 +509818,32 @@ property_value: curated_content_resource "https://www.malacards.org/card/mixed_s
 [Term]
 id: MONDO:0024359
 name: central sleep apnea due to periodic breathing
+def: "A sleep-related disorder characterized by cyclical breathing patterns where periods of shallow or absent breathing (apneas or hypopneas) alternate with periods of deeper breathing (hyperpnea). This cyclical pattern is caused by an instability in the body's respiratory control system, where the brain doesn't properly regulate breathing effort." [PMID:31604817]
 synonym: "central sleep apnea due to periodic breathing" EXACT []
+synonym: "central sleep apnea with Cheyne–Stokes breathing" NARROW [PMID:27920092]
+synonym: "Cheyne–Stokes breathing" NARROW [PMID:17296668]
 xref: ICD9:327.27 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: ICD9:786.09 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MEDGEN:783212 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: SCTID:85721000119105 {source="MONDO:equivalentTo"}
 xref: UMLS:C3662057 {source="MONDO:equivalentTo", source="MEDGEN:783212", source="MONDO:MEDGEN"}
-is_a: MONDO:0004731 ! central sleep apnea syndrome
+is_a: MONDO:0004731 {source="PMID:31604817"} ! central sleep apnea syndrome
 is_a: MONDO:0005087 {source="MONDO:Redundant"} ! respiratory system disorder
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8957" xsd:anyURI
 
 [Term]
 id: MONDO:0024360
 name: central sleep apnea caused by high altitude
 def: "A central sleep apnea syndrome induced by sleeping at an altitude higher than 2000 m. Central sleep apnea is caused by improper signaling from the brainstem to the respiratory muscles and is triggered by either hypoventilation or hyperventilation. Clinical signs include snoring, insomnia or hypersomnia, difficulty concentrating, and fatigue. Recurrent episodes of hypoxia/hypoxemia have long-term detrimental effects on cardiovascular health." [https://orcid.org/0000-0002-6581-7754, PMID:27343103]
-comment: Reason of obsoletion: duplicate. This will be merged with MONDO:0024359 central sleep apnea due to periodic breathing
-subset: obsoletion_candidate
-synonym: "central sleep apnea caused by high altitude" EXACT []
-synonym: "central sleep apnea due to high altitude" RELATED []
+comment: This term is not the same concept as central sleep apnea due to periodic breathing.
```

Comment:
I think you can expand a little bit. If someone doesn't know about the issue you are working on, it looks a bit weird. 
Maybe something like "This term is also called "central sleep apnea due to high-altitude periodic breathing". However, this term is not the same concept as central sleep apnea due to periodic breathing-MONDO:###(PMID###)